app/frontend/Calendar/DisplayEvent/Invitation.svelte

```diff
@@ -14,88 +14,67 @@
       {ex?.message ?? ex}
     {/await}
   {/if}
-  {#if message.scheduling == Scheduling.Request}
-    <hbox class="buttons">
+  <hbox class="buttons">
+    {#if selectedCalendar}
+      <AccountDropDown selectedAccount={selectedCalendar} accounts={calendars} withIcon={true} withLabel={true} disabled={calendars.length < 2} />
+    {/if}
+    {#if message.scheduling == Scheduling.Request}
       <Button
         label={$t`Confirm *=> Confirm to attend the meeting`}
         onClick={onAccept}
         icon={AcceptIcon}
         classes="accept" />
+      <Button
+        label={$t`Maybe *=> Maybe attend the meeting`}
+        onClick={onTentative}
+        icon={MaybeIcon}
+        classes="maybe secondary" />
       <Button
         label={$t`Reject *=> Decline to attend the meeting`}
         onClick={onDecline}
         icon={DeclineIcon}
         classes="decline secondary" />
-      <ButtonMenu buttonIcon={ChevronDownIcon}>
-        <MenuItem
-          label={$t`Maybe *=> Maybe attend the meeting`}
-          onClick={onTentative}
-          icon={MaybeIcon}
-          classes="maybe" />
-      </ButtonMenu>
-    </hbox>
```

Comment:
Please keep the UI as-is. We had intentionally put this into a menu.

Reply:
This was the result of a merge conflict. At least I got the right label.

Reply:
So, could you please restore it?

Reply:
Please also check whether there were other "merge errors".